src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/Intervention.kt

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity
+
+import org.hibernate.annotations.CreationTimestamp
+import java.time.OffsetDateTime
+import java.util.UUID
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.Id
+
+@Entity
+data class Intervention(
+  @Id @GeneratedValue val id: UUID? = null,
+  @CreationTimestamp val createdAt: OffsetDateTime? = null,
+
+  val name: String? = null
```

Comment:
i think this should be a required field

Reply:
yes agree - newer version it is also called title.  I'm thinking all of these should be non-null too.  There are additional fields in the newer version